components/measurement/CommonDetails.js

```diff
@@ -113,7 +114,7 @@ const CommonDetails = ({
         <DetailsBox
           collapsed={false}
           title={
-            <Flex px={3} alignItems='center' bg={theme.colors.gray2}>
+            <Flex px={3} flexWrap='wrap' alignItems='center' bg={theme.colors.gray2}>
```

Comment:
If this is added to handle layout in smaller screens, I suggest using `flexDirection` and others. Refer how NavBar.js for examples.